docs/general/ddl/data-types.rst

```diff
@@ -82,28 +82,116 @@ A basic boolean type. Accepting ``true`` and ``false`` as values. Example::
     cr> drop table my_bool_table;
     DROP OK, 1 row affected (... sec)
 
+.. _character-data-types:
+
+Character Types
+===============
+
+The general-purpose character data types available in CrateDB.
+
+Columns of below listed character data types can also be analyzed. By default
```

Comment:
I think we can remove "Columns of below listed character data types can also be analyzed."

Reply:
it was not correct actually, changed it now and pushed the fixup.